searchlib/src/tests/tensor/direct_tensor_store/direct_tensor_store_test.cpp

```diff
@@ -61,11 +61,12 @@ TEST_F(DirectTensorStoreTest, heap_allocated_memory_is_tracked)
     store.store_tensor(make_tensor(5));
     auto mem_1 = store.getMemoryUsage();
     auto ref = store.store_tensor(make_tensor(10));
-    size_t tensor_memory_used = store.get_tensor(ref)->count_memory_used();
+    auto tensor_memory_used = store.get_tensor(ref)->get_memory_usage();
```

Comment:
Consider rename _tensor_memory_used_ to e.g _tensor_memory_. Applies further down in file as well.